src/Prowlarr.Api.V1/swagger.json

```diff
@@ -766,13 +766,13 @@
                         "source": "ImportMechanismCheck",
                         "type": "warning",
                         "message": "Enable Completed Download Handling",
-                        "wikiUrl": "https://wiki.servarr.com/Prowlarr_System#Completed.2FFailed_Download_Handling"
+                        "wikiUrl": "https://wikijs.servarr.com/prowlarr/system#completed.2FFailed_Download_Handling"
```

Comment:
This isn't in Prowlarr is it? Appreciate that the swagger doc is probably out of date already though